pkg/controller/installation/validation_test.go

```diff
@@ -246,7 +246,10 @@ var _ = Describe("Installation validation tests", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	It("should prevent IPIP if BGP is disabled", func() {
+	// Previously, IPIP encapsulation were only possible with BGP enabled,
+	// however, Felix can do the same thing now which means IPIP with BGP disabled
+	// is supported.
+	It("should allow IPIP if BGP is disabled", func() {
```

Comment:
We can also remove these two tests, but given UTs are cheap, it might be good to keep them to not regress.